config/config.go

```diff
@@ -2118,3 +2118,47 @@ func ParseRemoteState(ctx context.Context, l log.Logger, opts *options.Terragrun
 
 	return cfg.GetRemoteState(l, opts)
 }
+
+func CleanStacks(_ context.Context, l log.Logger, opts *options.TerragruntOptions) error {
```

Comment:
I suspect this function can go to `config/stacks.go` to have all the stacks-related code in one place.